platooning_control_ihp/src/platoon_control_ihp.cpp

```diff
@@ -49,25 +49,19 @@ namespace platoon_control_ihp
         pnh_->param<double>("lowpassGain", config.lowpassGain, config.lowpassGain);
         pnh_->param<double>("lookaheadRatio", config.lookaheadRatio, config.lookaheadRatio);
         pnh_->param<double>("minLookaheadDist", config.minLookaheadDist, config.minLookaheadDist);
-
-        // Global params (from vehicle config)
         pnh_->getParam("/vehicle_id", config.vehicleID);
         pnh_->getParam("/vehicle_wheel_base", config.wheelBase);
-        pnh_->getParam("/control_plugin_shutdown_timeout", config.shutdownTimeout);
-        pnh_->getParam("/control_plugin_ignore_initial_inputs", config.ignoreInitialInputs);
 
         pcw_.updateConfigParams(config);
         config_ = config;
 
 	  	// Trajectory Plan Subscriber
-        //   TODO: update trajectory executor to call this plugin
-		trajectory_plan_sub = nh_->subscribe<cav_msgs::TrajectoryPlan>("PlatoonControlIHPPlugin/plan_trajectory", 1, &PlatoonControlIHPPlugin::trajectoryPlan_cb, this);
+		trajectory_plan_sub = nh_->subscribe<cav_msgs::TrajectoryPlan>("PlatooningControlPlugin/plan_trajectory", 1, &PlatoonControlIHPPlugin::trajectoryPlan_cb, this);
```

Comment:
If we're changing the name from Platoon... to Platooning..., then this message callback reference needs to change also.  Does this actually compile?